### PR TITLE
ansario/add-hello-world

### PR DESCRIPTION
**Pull request by ansario**
I added the hello world file.

### CODE REVIEW

File: hello.md [hunk 1]
```diff
@@ -0,0 +1 @@
+## hi emma
```
Comment (huntrontrakkr) [inline hello.md]: You should cap out names

**Review by huntrontrakkr — CHANGES_REQUESTED** [review]
See comments

File: hello.md (hunk 1 above)
Comment (ansario) [reply hello.md]: Thanks @huntrontrakkr ! Fixed!

**Review by huntrontrakkr — APPROVED** [review]
:shipit:

**Pull request merged**